OOBinarySearchTreeTord/es6-test/bst_test.js

```diff
@@ -0,0 +1,68 @@
+import BinarySearchTree from '../dist/js/BinarySearchTree';
+import { expect } from 'chai';
+
+describe('BinarySearchTree', function() {
```

Comment:
You're not using our test data. Make sure that insert works correctly without using untested functions.